site/realm/functions/onIncidentUpdate.js

```diff
@@ -14,10 +14,18 @@ exports = async function (changeEvent) {
     return;
   }
 
+  const notificationWhitelistedFields = [`title`, `description`, `date`, `Alleged deployer of AI system`, `Alleged developer of AI system`, `Alleged harmed or nearly harmed parties`, `reports`, `editors`, `editor_notes`];
```

Comment:
I took more time to look at the Realm configurations, and we actually already have a way of triggering in Realm based on the updated fields. We ignore the tSNE and embedding updates, for example:
https://github.com/responsible-ai-collaborative/aiid/blob/91ab98aba974da67b2aedf921e430d4f9d2caf73/site/realm/triggers/onIncidentUpdate.json#L11

Overall, though, I like the logic living in a function we can test, rather than some configuration file we can't test as easily.

What do you think?
Also curious for @cesarvarela.

Reply:
Sadly, we have to use configurations for now because no matter how careful you are with the function, the trigger still crashes if there are too many updates, like in the case of migrations. Filtering through configs avoids function execution altogether.